programs/client/TestHint.cpp

```diff
@@ -0,0 +1,116 @@
+#include "TestHint.h"
+
+#include <sstream>
+#include <iostream>
+#include <Common/Exception.h>
+#include <Common/ErrorCodes.h>
+#include <Parsers/Lexer.h>
+
+namespace DB
+{
+
+namespace ErrorCodes
+{
+    extern const int CANNOT_PARSE_TEXT;
+}
+
+}
+
+namespace
+{
+
+int parseErrorCode(std::stringstream & ss) // STYLE_CHECK_ALLOW_STD_STRING_STREAM
```

Comment:
I don't like stringstream,
using ReadBufferFromString will be more easy and less error-prone.

Reply:
Ok.